pkg/models/devops/devops.go

```diff
@@ -292,7 +292,8 @@ func (d devopsOperator) ListPipelineObj(projectName string, filterFunc PipelineF
 		})
 	}
 
-	var result []interface{}
+	// make sure the items be [] instead of nil when total items is equal to 0
+	var result = make([]interface{}, 0)
```

Comment:
It'll not happend. See also 
![image](https://user-images.githubusercontent.com/1450685/124909237-cb554000-e01c-11eb-94f6-ba48b682b76b.png)

Reply:
Real case:

![image](https://user-images.githubusercontent.com/16865714/125323973-fb307a80-e371-11eb-8c7b-1499e25cf6d9.png)

Reply:
I guess that's probably because `result[0:0]` might optimized by go compiler as `[]`.